demo-terraform/aws/serverless-app/dynamodb.tf

```diff
@@ -2,29 +2,37 @@
 
 ## Dynamodb Table
 resource "aws_dynamodb_table" "basic-dynamodb-table" {
```

Comment:
**Accurics remediation**
 
**[MEDIUM]: Ensure KMS customer managed keys are used for server side encryption (SSE) of AWS DyanamoDB tables**
 
**Vulnerability**
AWS DynamoDb tables do not use KMS customer managed keys for server side encryption (SSE).
 
**Remediation**
Server Side Encryption can be enabled for AWS DynamoDB tables. 
 In AWS Console - 
 1. Sign in to AWS Console and go to the AWS DynamoDB table.
 2. Select Tables in the navigation pane.
 3. Select the necessary table.
 4. Find Encryption Type under Table details in the Overview tab.
 5. Select Manage Encryption and select KMS.
 6. Click Save. 
 In Terraform - 
 Set 'server_side_encryption.enabled' to 'true' and set 'server_side_encryption.kms_key_arn' to a valid AWS KMS Key ARN.

source: accurics-